padle/view/globalstats_view.py

```diff
@@ -0,0 +1,92 @@
+from discordmenu.embed.components import EmbedMain, EmbedThumbnail
+from discordmenu.embed.view import EmbedView
+from tsutils.menu.components.config import UserConfig
+from tsutils.menu.components.footers import embed_footer_with_state
+from tsutils.menu.view.view_state_base import ViewStateBase
+from tsutils.tsubaki.links import MonsterImage
+from tsutils.tsubaki.monster_header import MonsterHeader
+
+
+class GlobalStatsViewState(ViewStateBase):
+    VIEW_STATE_TYPE: str = "GlobalStatsView"
+
+    def __init__(self, original_author_id, menu_type, raw_query="", current_day=0, num_days=0,
+                 extra_state=None, reaction_list=None, monster=None, stats=[]):
```

Comment:
need to add a QuerySettings here too so we can  have pretty colors for donors :)